Tetfuza/TetfuzaCLI/SystemConsoleDisplay.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tetfuza;
+using Tetfuza.Interfaces;
+
+namespace TetfuzaCLI
+{
+    public class SystemConsoleDisplay : IDisplay
+    {
+        public void GameScreen(TetfuzaBackend gameState)
+        {
+            RedrawFrame();
+            Console.CursorVisible = false;
+
+            WriteText("Level: " + gameState.Level);
+            WriteText("Score: " + CommasInNumber(gameState.Score));
+            DrawNextPiece(gameState.AfterPiece);
+
+
+
+            var board = gameState.ToString();
+            // Convert game board to an array of strings 
+            // and convert all block characters to appear the same
+            string[] boardView = board.Replace(TetfuzaBoard.LOCKDOWN_CHAR, TetfuzaBoard.MOVING_CHAR).Split(")");
+            WriteText("-----------------------");
+            // Loop through array and have each line centered on the screen
+            for (int i = 0; i < boardView.Length - 1; i++)
+            {
+                int location = i;
+                //if (_cheatCode.ToLower() == "up") // A small Easter Egg to make the pieces fall up
+                //    location = boardView.Length - 2 - i;
+                WriteText("| " + boardView[location] + "|");
+            }
+            WriteText("-----------------------");
+            WriteText(" ");
+            WriteText("Z and X to rotate");
+            WriteText("Arrow keys to move (left, right, down)");
+        }
+
+        public void WriteText(string text)
```

Comment:
My PR has a new implementation of this method that’s simpler

Reply:
I’ll add it after this is merged

Reply:
Your string format implementation is now in the refactor